release-notes.txt

```diff
@@ -1,3 +1,232 @@
+--------
++ CCN-Lite as a ICN network stack
++ RELIC: efficient cryptography library
+* fix TLSF to compile with -pedantic
+
+Supported platforms
+-------------------
+Additional support for the following boards:
++ weio board with  NXP LPC11U34 (ARM Cortex-M0)
++ Silicon Labs Wireless Eval Kit SLWSTK6220A (Wonder Gecko)
++ STM32 Nucleo-F401
+
+Drivers
+-------
++ Arduino-mega2560 GPIO
++ Arduino pin mapping for Mega2560 and Due
+
+Network drivers
+---------------
++ enc28j60 Ethernet chip
++ at86rf2xx: Add support for channel page
+* at86rf2xx: fix LQI reading
+* implement sleep mode for at86rf2xx
+
+Sensors drivers
+---------------
++ AT30TSE75x temperature sensor
++ TCS3772 Color Light-to-Digital converter
+
+System libraries
+----------------
++ partial support for the Arduino API
++ lightweight semaphores
++ fmt: simple string formatting library
++ xtimer: 32-bit version of msg_recv_timeout
+* implicit socket binding for POSIX connect() and sendto()
+* posix_semaphore: make API POSIX compliant
+
+Examples
+--------
++ microcoap/conn example
++ minimal GNRC networking example
+
+Build System
+------------
+* split the Cortex-M0 buildtest group to avoid timeout issues with Travis
+* split the Cortex-M4 buildtest group to avoid timout issues with Travis
+
+Other
+-----
++ vagrant configuration
++ documentation: various high-level descriptions of crucial features
++ IoT-LAB: create and connect to debug server
+* pyterm: fix problems with German umlauts as input
+
+Fixed Issues from the last release
+==================================
+
+#2724: Add support for serial number passing to CMSIS boards, document it
+    Documentation about how to discover and set the serial number of CMSIS-DAP chips is missing
+#3201: Odd length packet snips cause invalid check sum
+    If an odd length packet snip occurs in a packet and is not the last snip
+    (in the order the packet is supposed to be, not in the list's order)
+    in a packet it will generate a wrong check sum.
+
+Known Issues
+============
+
+network related issues
+----------------------
+#3075: nhdp: unnecessary microsecond precision
+    NHDP works with timer values of microsecond precision which is not required. Changing
+    to lower precision would save some memory.
+#3086: Max. packet length for AT86RF2XX
+    The size of the link-layer header is not dynamically calculated, but instead the maximum#
+    size is always assumed.
+#3970: RPL: Advertise DODAG only over the assigned interface
+     gnrc_rpl seems to multicast DIOs over all interfaces, though gnrc_rpl_init expects an
+     interface as parameter and sets the RPL-nodes multicast address only for that interface.
+#4048: potential racey memory leak
+     According to the packet buffer stats, flood-pinging a multicast destination may lead to a
+     memory leak due to a race condition. However, it seems to be a rare case and a
+     completely filled up packet buffer was not observed.
+#4462: IPHC/NHC broken between RIOT nodes
```

Comment:
basic NHC between two RIOT nodes (and I think even between RIOT and Linux) works, so we should add a more detailed description what does not work